src/gerrit.js

```diff
@@ -95,6 +95,15 @@ export class Changelist {
     return this.json_.work_in_progress === true;
   }
 
+  // Returns whether the user is in the attention set of this CL.
+  needsAttention(user) {
+    // This object is an object with properties, rather than a true map
+    if(this.json_.hasOwnProperty("attention_set")) {
```

Comment:
nit: can you add space between "if" and open parenthese?

Reply:
Done.